tests/bud.bats

```diff
@@ -1894,6 +1902,21 @@ function _test_http() {
   expect_output --substring "COPY --from=build: no stage or image found with that name"
 }
 
+@test "bud with copy-from referencing the current stage and with --keep-ownership" {
+  _prefetch busybox
+  target=busybox-derived
+  run_buildah build $WITH_POLICY_JSON -t ${target} -f $BUDFILES/copy-from-keep-ownership/Dockerfile $BUDFILES/copy-from-keep-ownership
+  expect_output --substring "\-rw-r--r--    1 123      456"
+}
+
+@test "bud with copy-workdir and with --keep-ownership" {
+  _prefetch busybox
+  target=busybox-derived
+  chown 321:999 $BUDFILES/copy-from-keep-ownership/file2.txt
```

Comment:
You can't do this: first, the test may run rootless. Second, even if root, you would be chowning a file that you have no right to chown (possibly in the developer's source dir, or possibly in the rpm-managed `/usr/share/tests/buildah`.

Solution: add `skip_if_rootless` to this test, and copy the build context into `$TEST_SCRATCH_DIR`.